src/components/HighlightedPosts/index.module.scss

```diff
@@ -0,0 +1,112 @@
+@import "common/breakpoints";
+
+.root {
+  position: relative;
+
+  margin-bottom: 89px;
+}
+
+.nextArrow,
+.prevArrow {
+  position: absolute;
+  bottom: 0;
+  z-index: 1;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 56px;
+  height: 56px;
+
+  background: transparent;
+  border: 1px solid var(--c-blue-1);
+  border-radius: 100%;
+  cursor: pointer;
+}
+
+.nextArrow {
+  right: 0;
+}
+
+.prevArrow {
+  right: 72px;
+}
+
+.post {
+  display: flex;
+  flex-direction: column-reverse;
+  align-items: center;
+  padding-bottom: 72px;
+
+  @include media(">=tablet") {
+    display: grid;
+    grid-column-gap: 28px;
+    grid-template-columns: repeat(2, 1fr);
+  }
+}
+
+.details {
+  max-width: 442px;
+}
+
+.image {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  max-width: 442px;
+  padding: 36px 16px 26px;
+
+  @include media(">=tablet") {
+    padding-top: 16px 16px 0 0;
+  }
+}
+
+.cover {
+  width: 100%;
+  max-width: 100%;
+
+  border-radius: 20px;
+
+  transform: rotate(5deg);
+}
+
+.intro,
+.author {
+  display: block;
+  margin-bottom: 40px;
+}
+
+.author {
+  color: var(--c-blue-1);
+}
+
+.title {
+  margin-bottom: 40px;
+
+  font-family: "Acta", serif;
+  font-size: 56px;
+  font-weight: 400;
+  line-height: 120%;
+}
+
+.dots {
+  display: flex !important;
```

Comment:
Instead of using `!important` I believe you should be able to use `:global` with CSS modules when wanting to override external library styles

Reply:
Not in this case. The library itself allows me to specify a class to override its styles. I'm not supposed to reference its classes directly